wit/testdata_test.go

```diff
@@ -499,6 +504,43 @@ func TestHasBorrow(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
+	testCases := []struct {
```

Comment:
Can you pull this into a separate function, e.g. `TestHasBorrow` in abi_test.go?

Reply:
Done